### PR TITLE
Update dependency references

### PR DESCRIPTION
**Pull request by mtrupkin**
Log configuration changes were made in the sb11-shared-code project.
Updating Student Report Processor to include these changes.
SRP depends on sb11-shared-code transitively via tds-dll-common and tds-dll-mysql

### CODE REVIEW

**Comment by millerts** [comment]
Shouldn't this be going into the develop branch not master?

**Comment by jtreuting** [comment]
yes, i just switched to develop

File: shared-log4j/pom.xml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-  <name>Log4j utilities</name>
-  <description>Utilities to extend Log4j logging framework</description>
-  <dependencies>
-	<dependency>
-		<groupId> log4j</groupId>
-		<artifactId>log4j</artifactId>
-		<scope>provided</scope>
-	</dependency>
-  </dependencies>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (millerts) [inline shared-log4j/pom.xml:1]: Not sure if this is intentional, but the main change here looks to be going from `SNAPSHOT` to no snapshot.  However, it seems the entire file was formatted which is why we're seeing full red/green on the change.  We've been avoiding reformatting files on other changes to legacy.  Not sure if it matters here or the other files that were reformatted.

**Review by millerts — APPROVED** [review]
A couple files look like they were reformatted.  Not sure how much of that we should be doing.

**Review by millerts — CHANGES_REQUESTED** [review]
I think you may be on the master branch locally which is the reason you needed to flip these to snapshot.  For example, isn't this the same as what you've done [develop master pom.xml](https://github.com/SmarterApp/SS_SharedMultiJar/blob/develop/pom.xml#L113)

**Pull request closed without merging**